### ISSUE TITLE
Patch size for fit_generator and validation_generator

### ISSUE THREAD

**Issue by majd1988shahin** [keras-team/keras#13678]
Hallo ,
i have data set with 1000 image
i am intending to use 800 for train and 200 for validation 

that mean 80 steps in epoche
but how match is Patch_size in validation generator ???
____________________________

g=Generator(Patch_size=10,start=0,end=800)

gv=Generator(Patch_size=10,start=800,end=1000) 
model=creat_Model()

optamizer=keras.optimizers.Adam(lr=0.00002, beta_1=0.9, beta_2=0.999, amsgrad=False)
model.compile(optimizer=optamizer,loss=loss) 
h=model.fit_generator(g,validation_data=gv,steps_per_epoch=800/10,epochs=200,use_multiprocessing=True)
